File: demo/src/main/java/com/example/city/demo/entities/City.java
package com.example.city.demo.entities;

import lombok.*;

import java.util.UUID;

@Setter
@Getter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class City {

    private UUID cityId;
    private String cityName;
    private int population;
    private int totalFamilies;

}

File: cityshow/lib/models/city.dart
class City {
  final String cityId;
  final String cityName;
  final int population;
  final int totalFamilies;

  City({
    required this.cityId,
    required this.cityName,
    required this.population,
    required this.totalFamilies,
  });

  Map<String,dynamic> toJson(){
    return {
      'cityId':this.cityId,
      'cityName':this.cityName,
      'population':this.population,
      'totalFamilies':this.totalFamilies,
    };
  }

  factory City.fromJson(Map<String, dynamic> json) {
    return City(
      cityId: json['cityId']??'',
      cityName: json['cityName']??'unknown',
      population: json['population']??0,
      totalFamilies: json['totalFamilies']??0,
    );
  }
}
